src/components/Banner/Banner.js

```diff
@@ -9,7 +9,8 @@ const Banner = () => {
     <div aria-label="banner" className={banner}>
       <p className={bannerText}>
         <strong>
-          Carbon v10 is in maintenance mode. Support will end on Sept. 30, 2024.
+          Carbon v10 reached end of support on September 30, 2024 and will not
```

Comment:
Just a grammar question but also not really sure what is right, it just reads kinda funny. Not a blocker though really so I'll go ahead and approve. 

Should "end of support" be a hyped word "**end-of-support**" 
OR "Carbon v10 reached **the** end of support on Septemeber..."